index.html

```diff
@@ -82,6 +82,225 @@ <h1>
           </ul>
         </div>
       </section>
+      <section id="work-section">
+        <article class="work-container flex">
+          <div class="work-img img-one"></div>
+          <div class="work-description">
+            <h3>Tonic</h3>
+            <div class="client-des">
+              <ul class="client-name">
+                <li>CANOPY</li>
+              </ul>
+              <ul>
+                <li>Back End Dev</li>
+              </ul>
+              <ul>
+                <li>2015</li>
+              </ul>
```

Comment:
@chaw-bot  I am assuming they are different categories so that is why I am trying to put them in separate  list in case we have future integration of let's say back end and front end dev also

Reply:
Okay noted !!